src/pymortests/operators.py

```diff
@@ -292,21 +292,28 @@ def test_restricted(operator_with_arrays):
 def test_InverseOperator(operator_with_arrays):
     op, mu, U, V = operator_with_arrays
     inv = InverseOperator(op)
+    if is_windows_platform() and 'Block' in op.__class__.__name__:
+        rtol = atol = 2e-13
```

Comment:
Could you find out which `operator_with_arrays` is the culprit? I would agree to raise the tolerance for all platforms. But before that I would like to know what exactly is happening.